src/components/navBar/NavBar.js

```diff
@@ -21,6 +22,12 @@ class NavBar extends Component {
     });
   };
 
+  taskView() {
```

Comment:
I'm not too sure what this is doing, it seems that the button is wrapped in an anchor tag which takes us to the archive, so the onclick of the button isn't doing anything.

Reply:
Ok, I'll look at it now : (